testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -3886,3 +3892,778 @@ fn antientropy_integration_test() {
     test_observer::clear();
     channel.stop_chains_coordinator();
 }
+
+fn wait_for_mined(
+    btc_regtest_controller: &mut BitcoinRegtestController,
+    blocks_processed: &Arc<AtomicU64>,
+    http_origin: &str,
+    users: &[StacksPrivateKey],
+    account_before_nonces: &Vec<u64>,
+    batch_size: usize,
+    batches: usize,
+    index_block_hashes: &mut Vec<StacksBlockId>,
+) {
+    let mut all_mined_vec = vec![false; batches * batch_size];
+    let mut account_after_nonces = vec![0; batches * batch_size];
+    let mut all_mined = false;
+    for _k in 0..10 {
+        next_block_and_wait(btc_regtest_controller, &blocks_processed);
+        sleep_ms(10_000);
+
+        let (ch, bhh) = get_chain_tip(http_origin);
+        let ibh = StacksBlockHeader::make_index_block_hash(&ch, &bhh);
+
+        if let Some(last_ibh) = index_block_hashes.last() {
+            if *last_ibh != ibh {
+                index_block_hashes.push(ibh);
+                eprintln!("Tip is now {}", &ibh);
+            }
+        }
+
+        for j in 0..batches * batch_size {
+            let account_after = get_account(&http_origin, &to_addr(&users[j]));
+            let account_after_nonce = account_after.nonce;
+            account_after_nonces[j] = account_after_nonce;
+
+            if account_before_nonces[j] + 1 <= account_after_nonce {
+                all_mined_vec[j] = true;
+            }
+        }
+
+        all_mined = all_mined_vec.iter().fold(true, |acc, elem| acc && *elem);
+        if all_mined {
+            break;
+        }
+    }
+    if !all_mined {
+        eprintln!(
+            "Failed to mine all transactions: nonces = {:?}, expected {:?} + {}",
+            &account_after_nonces, account_before_nonces, batch_size
+        );
+        panic!();
+    }
+}
+
+#[test]
+#[ignore]
+fn atlas_stress_integration_test() {
```

Comment:
This test doesn't appear to work correctly in the Github action.

Reply:
I think there's an issue line 4630, when retrieving the expected content hash - it looks like the tuple for retrieving the content hash would be invalid.
Do you want me to take over?

Reply:
I think I've fixed it, but let's see if the CI takes it.